Code/Framework/AzCore/Tests/Math/QuaternionTests.cpp

```diff
@@ -523,4 +524,143 @@ namespace UnitTest
         const float angle = absQuat.GetEulerRadians().GetX();
         EXPECT_THAT(angle, testing::FloatEq(-AZ::Constants::HalfPi));
     }
+
+    struct EulerTestArgs {
+        AZ::Vector3 euler;
+        AZ::Quaternion result;
+    };
+
+    using AngleRadianTestFixtureXYZ = ::testing::TestWithParam<EulerTestArgs>;
+
+    TEST_P(AngleRadianTestFixtureXYZ, EulerRadiansXYZ) {
+        auto& param = GetParam();
+        EXPECT_THAT(AZ::Quaternion::CreateFromEulerRadiansXYZ(param.euler), IsClose(param.result));
+    }
+
+     TEST_P(AngleRadianTestFixtureXYZ, EulerDegreesXYZ) {
+        auto& param = GetParam();
+        EXPECT_THAT(AZ::Quaternion::CreateFromEulerDegreesXYZ(Vector3RadToDeg(param.euler)), IsClose(param.result));
+    }
```

Comment:
coding standards: Brace placement again `{` (easy to forget I know, clang-format may help here, especially if you can apply it to a selection)

Reply:
(repeated below)